torchaudio/functional/functional.py

```diff
@@ -1429,6 +1505,10 @@ def resample(
 ) -> Tensor:
     r"""Resamples the waveform at the new frequency using bandlimited interpolation.
 
+    .. devices:: CPU CUDA
```

Comment:
could you move this down under the link below?

Reply:
[PEP257](https://peps.python.org/pep-0257/#multi-line-docstrings
) recommends multi-line-docstrings to be split into one-line summary and elaborated explanation.

I would like to put badges between the summary and the rest, and the URL does not fit nicely.

Can I add some more explanation along the URL?

Reply:
The homepage has an example to cite the page. https://ccrma.stanford.edu/~jos/resample/resample-citation.html

I moved it to bibtex reference. How does it look?

Reply:
yup that looks good